PMM-12985 Apply v3 API infra modifications and update docs (#3207)
percona/pmm: docs/api/inventory/overview.md

---
title: Overview
slug: pmm-inventory
categorySlug: inventory-api
hidden: 0
order: 0
---

> This section will provide you with information on how to add, update or remove services, nodes and agents.

Reviewer: ```suggestion
This section guides you on how to add, update, or remove services, nodes, and agents.
```
